src/main/java/com/denizenscript/clientizen/objects/ModTag.java

```diff
@@ -0,0 +1,251 @@
+            return new ElementTag(object.getMetadata().getId(), true);
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.display_name>
+        // @returns ElementTag
+        // @description
+        // Returns a mod's display name.
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "display_name", (attribute, object) -> {
+            return new ElementTag(object.getMetadata().getName(), true);
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.description>
+        // @returns ElementTag
+        // @description
+        // Returns a mod's description.
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "description", (attribute, object) -> {
+            return new ElementTag(object.getMetadata().getDescription(), true);
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.version>
+        // @returns ElementTag
+        // @description
+        // Returns a mod's version.
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "version", (attribute, object) -> {
+            return new ElementTag(object.getMetadata().getVersion().getFriendlyString(), true);
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.authors>
+        // @returns MapTag
+        // @description
+        // Returns a mod's authors, as a map of author name to their contact information.
+        // The contact information is a map of contact platforms to identification on that platform.
+        // Note that mods can provide anything here, although most mods will obviously provide valid info.
+        // -->
+        tagProcessor.registerStaticTag(MapTag.class, "authors", (attribute, object) -> {
+            return Utilities.personsToMap(object.getMetadata().getAuthors());
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.contributors>
+        // @returns MapTag
+        // @description
+        // Returns a mod's contributors, as a map of contributor name to their contact information.
+        // The contact information is a map of contact platforms to identification on that platform.
+        // Note that mods can provide anything here, although most mods will obviously provide valid info.
+        // -->
+        tagProcessor.registerStaticTag(MapTag.class, "contributors", (attribute, object) -> {
+            return Utilities.personsToMap(object.getMetadata().getContributors());
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.contact_info>
+        // @returns MapTag
+        // @description
+        // Returns a mod's contact information, as a map of contact platforms to identification on that platform.
+        // Note that mods can provide anything here, although most mods will obviously provide valid info.
+        // -->
+        tagProcessor.registerStaticTag(MapTag.class, "contact_info", (attribute, object) -> {
```

Comment:
There are some common ones like `website` and `repo`, Quilt has `homepage`, `issues`, `sources`

Reply:
Added a few common ones to the meta